Spanner/src/SpannerClient.php

```diff
@@ -193,6 +193,8 @@ class SpannerClient
      *           {@see \Google\Cloud\Spanner\V1\DirectedReadOptions}
      *           If using the `replicaSelection::type` setting, utilize the constants available in
      *           {@see \Google\Cloud\Spanner\V1\DirectedReadOptions\ReplicaSelection\Type} to set a value.
+     *     @type bool $xGoogSpannerRouteToLeader Enable/disable Leader Aware Routing.
```

Comment:
Since this is a client option, it doesn't have to exactly word the header. Probably something like `routeToLeader` works better?

Reply:
Sounds good, changed